documentation/plain_rules.md

```diff
@@ -747,7 +752,7 @@ respectively.
 The formats are
 
 -   [NGSIv1 deprecated format](https://fiware-orion.readthedocs.io/en/1.15.1/user/geolocation/index.html#defining-location-attribute)
--   [NGSIv2 current format](http://telefonicaid.github.io/fiware-orion/api/v2/stable/), section "Geospatial properties
+-   [NGSI v2 current format](http://telefonicaid.github.io/fiware-orion/api/v2/stable/), section "Geospatial properties
```

Comment:
Just curiosity but :)... why "NGSI v2" is preferred over "NGSIv2"? Is is due to some English language reason? Maybe is some kind of FIWARE-wide documentats homogenization?

Reply:
** English language reason:** This is a pair of acronyms place next to each other is **NGSI** (_"Next Generation Service Interface"_, which includes NGSI v1, NGSI v2 and NGSI LD) . **v1**, **v2** , **LD** etc. is a second acronym and hence is treated as a separate word. It is common English lexicography to separate acronyms with a space rather than a run-on word.

_NGSIv2_ just looks weird to me as a native.

Reply:
Ok. Let's go with "NGSI v2" :)

NTC